packages/venia-concept/src/components/Checkout/receipt.js

```diff
@@ -12,17 +13,33 @@ class Exit extends Component {
             footer: string,
             root: string
         }),
-        resetCheckout: func.isRequired
+        resetCheckout: func.isRequired,
+        orderId: integer
     };
 
+    static defaultProps = {
+        //TODO: implement fetching of orderId
+        orderId: 777
+    }
+
     render() {
-        const { classes, resetCheckout } = this.props;
+        const { classes, resetCheckout, orderId } = this.props;
 
         return (
             <div className={classes.root}>
-                <div className={classes.body}>Thank you for your order!</div>
-                <div className={classes.footer}>
+                <div className={classes.body}>
+                    <h2 className={classes.headerText}>
```

Comment:
We do not have css for this className. In general, we need to discuss our approach with classify HOC.